app/styles/ui/check-runs/_ci-check-run-logs.scss

```diff
@@ -74,28 +72,34 @@
 
         .line {
           display: flex;
+          width: 100%;
 
           .line-number {
             overflow: hidden;
             text-overflow: ellipsis;
             text-align: right;
             display: inline-block;
+            flex-shrink: 0;
+            flex-grow: 0;
           }
 
           .line-content {
             margin-left: var(--spacing);
-            white-space: nowrap;
-            max-width: 90%;
+            word-break: normal;
+            overflow-wrap: anywhere;
```

Comment:
I'd say this deserves a comment to at least indicate this is the only way we found to keep a sane line break for "normal" text, but also for long paths in the logs.